apps/meteor/client/lib/getRoomTypeTranslation.ts

```diff
@@ -0,0 +1,18 @@
+import type { IRoom } from '@rocket.chat/core-typings';
+
+import { t } from '../../app/utils/lib/i18n';
+
+export const getRoomTypeTranslation = (type: IRoom['t']) => {
```

Comment:
What about adding a unit test to cover this function?

Reply:
Aligned this with Marcos, the function is pretty basic and simple, doesn't need a test for this function.